code/datums/components/suit_autodoc.dm

```diff
@@ -62,10 +85,16 @@
 	src.brute_chems = brute_chems || default_brute_chems
 	src.tox_chems = tox_chems || default_tox_chems
 	src.pain_chems = pain_chems || default_pain_chems
-	
+
 	if(!isnull(overdose_threshold_mod))
 		src.overdose_threshold_mod = overdose_threshold_mod
 
+	if(!isnull(supporting_limbs))
```

Comment:
supporting limbs is a feature of hardsuits in other codebases, i feel like it shouldnt be a feature of the suit autodoc component